lib/ansible/plugins/callback/__init__.py

```diff
@@ -227,7 +227,8 @@ def _get_item(self, result):
             item = result.get('_ansible_item_label')
         else:
             item = result.get('item', None)
-
+        #Fix to output string of items rather than unicode of items.
+        item = [str(x) for x in item]
```

Comment:
A couple of things:
 - you are assuming item is a list, it can be any type
 - str() is not a good function across versions of python, that is why we have to_text and to_native